package.json

```diff
@@ -8,11 +8,11 @@
     "node": ">=12"
   },
   "scripts": {
-    "watch:webpack": "NODE_ENV=development npx webpack-cli -w",
+    "watch:webpack": "npx cross-env NODE_ENV=development webpack-cli -w",
```

Comment:
or we can use `--node-env` to provide `process.env.NODE_ENV` (https://webpack.js.org/api/cli/#node-env) or use  the `--mode` flag (https://webpack.js.org/api/cli/#flags)
what do you think?

Reply:
👆 @z-x

Reply:
@nemvalid - I have no experience using it and not much time to test it on all the platforms, so if you say it works, I'll gladly approve a PR. :)

Reply:
@z-x I'm not use what you mean. You have no experience with webpack or passing parameters? Would you mind checking it please? (Another link about handling environment variables with webpack https://webpack.js.org/guides/environment-variables/)

Reply:
@z-x which platform do you use? I'm happy to check it on my mac and I think Gabor has a windows dev environment already set up. We can also ask QA folks if needed.

Reply:
Any updates on this one? Did you manage to figure it out?